govdelivery/api.py

```diff
@@ -130,3 +130,29 @@ def set_subscriber_answers_to_question(self, contact_details, question_id, answe
 
         payload = xml_payloads.free_response_to_question(question_id_encoded, answer_text)
         return self.call_api(path, "put", payload)
+
+    set_subscriber_answer_to_free_response_question = set_subscriber_answers_to_question
+
+    def set_subscriber_answer_to_select_question(
+        self,
+        contact_details,
+        question_id,
+        answer_id
+    ):
+        subscriber_id = base64.b64encode(contact_details)
+        question_id_encoded = base64.b64encode(question_id)
+        answer_id_encoded = base64.b64encode(answer_id)
+
+        path = self.translate_path(
+            '/api/account/$account_code/subscribers/$subscriber_id/questions/'
```

Comment:
Minor style thing: at first read, while valid, the breaking of the string on two lines like this is unexpected compared with the other arguments to the method call. I might more clearly separate this out like

```py
path = self.translate_path(
    (
        '/api/account/$account_code'
        '/subscribers/$subscriber_id'
        '/questions/$question_id_encoded/responses.xml'
    ),
    subscriber_id=subscriber_id,
    ...
)

Reply:
I took it one step further, hope that's OK :)